decisions/community-forum-platform.md

```diff
@@ -0,0 +1,349 @@
+
+### Conclusions about using stackexchange
+
+* Great tool for Q&A knowledge base but explicitly NOT for building community & connections
+* If we used it, we would have to maintain some _other_ forum for open-ended community discussions.
+
+## Proposal: Use Discourse for All Community Support
+
+Related Github Issues:
+* **original discussion** Discourse - Better community building https://github.com/ipfs/community/issues/69
+* Discourse: Evaluation Checklist https://github.com/ipfs/community/issues/213
+* `*.ipfs.io` domain for discourse https://github.com/ipfs/ops-requests/issues/33
+* Try out discourse.org for Community Forums  https://github.com/ipfs/community/issues/190
+* Merge ipfs/ipfs, ipfs/community and ipfs/pm https://github.com/ipfs/community/issues/191
+
+### Strengths of using discourse
+* Would be a one-stop landing place for anyone who needs to get answers, find help, share their excitement, tell other people about their projects, etc.
+* Doubles as a mailing list, which would allow us to decommission the very-low-traffic, oft-forgotten google group
+* Very good search -- easy for people to find existing discussions and answers to questions
+* Allows us to consolidate ipfs/faq, ipfs/support and parts of ipfs/ipfs repositories into one platform (REDUCTO!✨)
+* notifications can be configured to suit wide variety of preferences
+* You can curate threads to emphasize things like important topics, common questions, especially good answers, etc.
+* Provides ways to give community members a range of powers and responsibilities
+
+### Drawbacks of using discourse
+
+@jbenet's [personal opinion on discourse itself](https://github.com/ipfs/community/issues/69#issuecomment-158861396):
+> - every discourse site i've participated in promotes "the act of posting messages" over "the act of communicating ideas", so in my experience the tends to devolve into _very low_ [SNR](https://en.wikipedia.org/wiki/Signal-to-noise_ratio).
+> - to me, the UI/UX feels very hostile. _i personally_ won't use it. feel free to have one, but i won't be there.
+> - discourse is yet-another centralized comm platform. i would like to have something distributed
+> - it's another damn silo!! -- i'm sick and tired of information silos that hold our data hostage! Our comm data belongs on the web itself, and this is what ipfs is for! _i can't wait_ till we have something _as good as github issues_ so we can ditch it for the web itself! /rant.
+
+General points:
+* would require a lot more management of community discussions in order to maintain a good [SNR](https://en.wikipedia.org/wiki/Signal-to-noise_ratio). Hopefully a lot of that effort can come from community members
+* would require attention to keep "community discussion" on discourse from drifting away from task-oriented, github-based discussion about implementation, maintenance and design
+* would be a protocol change for core contributors
+* would add to the already-crushing volume of notifications
```

Comment:
Actually, it could DECREASE the noise, see https://github.com/ipfs/community/pull/246#discussion_r105844087